From 73d0c6707d761c8453da99fd917bfd46805c3e87 Mon Sep 17 00:00:00 2001
From: Eric Paris <eparis@redhat.com>
Date: Thu, 4 Sep 2014 11:30:38 -0400
Subject: [PATCH 5/6] Do not ignore local GOPATH, ignore 3rd party deps

---
 hack/config-go.sh | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/hack/config-go.sh b/hack/config-go.sh
index 92dda10..f5fc420 100644
--- a/hack/config-go.sh
+++ b/hack/config-go.sh
@@ -105,9 +105,16 @@ kube::setup_go_environment() {
     fi
   fi
 
-  # Set GOPATH to point to the tree maintained by `godep`.
-  GOPATH="${KUBE_TARGET}:${KUBE_REPO_ROOT}/Godeps/_workspace"
+  LOCALGOPATH="${KUBE_TARGET}"
+  if [ -z ${GOPATH-} ]; then
+    GOPATH=${LOCALGOPATH}
+  else
+    GOPATH=${GOPATH}:${LOCALGOPATH}
+  fi
   export GOPATH
+  # Set GOPATH to point to the tree maintained by `godep`.
+  #GOPATH="${KUBE_TARGET}:${KUBE_REPO_ROOT}/Godeps/_workspace"
+  #export GOPATH
 
   # Unset GOBIN in case it already exsits in the current session.
   unset GOBIN
@@ -174,4 +181,3 @@ export KUBE_GO_PACKAGE
   # TODO: This symlink should be relative.
   ln -s "${KUBE_REPO_ROOT}" "${go_pkg_dir}"
 )
-
-- 
1.9.3

